### PR TITLE
Update Pneumatics Code to Use Spikes

### LINKED ISSUE #177: Spike H-Bridge based pneumatics
We need to implement a pneumatics system based on Spike H-Bridge relays instead of the PCM, since apparently the PCM hates us. Hopefully we don't also let the magic smoke out of a Spike.

We will have:
- One Spike wired such that when it is powered on the cylinders extend
- One Spike wired such that when it is powered on the cylinders retract
- One Spike wired to the compressor
- One digital pressure switch connected to the DIO port on the robot. The compressor should be powered ON when the signal pin of this circuit is shorted to ground. (i.e. when the DIO object reports true)

We should probably just implement a completely separate compressor subsystem that has a default command to handle its relay control. The end() of this command should turn of the compressor so that when we disable, the compressor stops.

This is our top priority. If possible, it would be great to have this done before the meeting today.

### PR DESCRIPTION
**Pull request by quackitsquinn**
Updates the code for pneumatics to use Spike relays instead of a PCM. Closes #177

### CODE REVIEW

**Comment by willitcode** [comment]
Tag the issue number

**Review by willitcode — CHANGES_REQUESTED** [review]
Looks good except for one minor comment. @Berdenson please review for any JavaDoc nonsense that I missed

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -99,6 +101,8 @@ public class RobotContainer {
   /** Singleton instance of the intake {@link Camera} for the whole robot. */
   public static Camera intakeCamera = new Camera(Constants.Camera.INTAKE_CAMERA_ID);
 
+  public static Compressor compressor = new Compressor();
```
Comment (Berdenson) [inline src/main/java/frc/robot/RobotContainer.java:104]: this needs javadoc

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -257,6 +261,8 @@ public class RobotContainer {
   /** Singleton instance of {@link ZeroGyro} for the whole robot. */
   public static ZeroGyro zeroGyro = new ZeroGyro();
 
+  public static ControlCompressor controlCompressor = new ControlCompressor();
```
Comment (Berdenson) [inline src/main/java/frc/robot/RobotContainer.java:264]: this also needs javadoc

**Review by Berdenson — CHANGES_REQUESTED** [review]
javadocccccc

**Review by Berdenson — APPROVED** [review]
yea looks good

**Review by willitcode — APPROVED** [review]

**Pull request merged**